ncl/ncl_entries/sin_trigonometry.ipynb

```diff
@@ -0,0 +1,282 @@
+{
```

Comment:
<p>Why is this part here at the bottom?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>